files/es/_redirects.txt

```diff
@@ -1249,7 +1249,7 @@
 /es/docs/MDN/Guidelines/Project:Guía_de_estilo	/es/docs/MDN/Writing_guidelines/Writing_style_guide
 /es/docs/MDN/Guidelines/Writing_style_guide	/es/docs/MDN/Writing_guidelines/Writing_style_guide
 /es/docs/MDN/Kuma	/es/docs/MDN/Community/Contributing/Our_repositories
-/es/docs/MDN/Kuma/Introduction_to_KumaScript	/es/docs/orphaned/MDN/Tools/KumaScript
+/es/docs/MDN/Kuma/Introduction_to_KumaScript	https://github.com/mdn/yari/tree/main/docs/kumascript
```

Comment:
@Graywolf9  just to understand this change, why do you add the full host instead of the relative path as other files do?

Reply:
The `en-US` doc did the same [here](https://github.com/mdn/content/pull/22638/files#diff-afe010c4707d10d488b1e1c050b3298da03267dec0abca46d6d444b2ae8774a5) on the added line 5545.

Reply:
Gracias por la revisión! @JuanVqz 

Justo como menciona yarusome, y como cite en la descripción, estoy haciendo el cambio como se hizo en `en-Us`

Por otra parte es por que las rutas relativas son para la documentación MDN y ésta es una redirección al respositorio de Yari.